src/api.ts

```diff
@@ -199,3 +206,29 @@ export function toEditorConfig(options: TextEditorOptions) {
 		return tabSize === 'auto' ? 4 : parseInt(String(tabSize), 10)
 	}
 }
+
+/**
+ * Get the URI of the .editorconfig file in the given folder
+ */
+export async function getEditorConfigUri(folderUri: Uri) {
```

Comment:
```suggestion
export async function getEditorConfigUri(folderUri: Uri): string | null {
```

It's clearer since `null` implies we know there's nothing. You can then replace `return` below with `return null`.

Reply:
I followed the existing code style, which doesn't include explicit return types

Reply:
Sorry for being unclear. You can omit the return type. The main point is to return `null` instead of `undefined`.